src/chat/_chatwidget.py

```diff
@@ -224,6 +224,45 @@ def join(self, channel):
     def log_event(self, e):
         self.serverLogArea.appendPlainText("[%s: %s->%s]" % (e.eventtype(), e.source(), e.target()) + "\n".join(e.arguments()))
 
+    def updateAutojoinIRCMenu(self):
+        self.client.menuAutojoinIRC.clear()
+        
+        autojoin_channels = Settings.get("chat/autojoin_channels", [], type=str).split(".")
```

Comment:
The default type and the type argument given by `type=str` should match. So default should be `""`, otherwise `[].split(".")` will fail.